src/coreclr/md/compiler/mdutil.cpp

```diff
@@ -242,11 +243,12 @@ HRESULT LOADEDMODULES::FindCachedReadOnlyEntry(
         // The cache is locked for read, so the list will not change.
 
         // Figure out the size and timestamp of this file
```

Comment:
What is this cache used for? Do we still need it? Would it be better to delete it?

Reply:
It's called by `IMetaDataDispenser::OpenScope` and reusing the same object for same file. I don't get a good insight for its impact and performance of creating new `RegMeta` instance.